cves/CVE-2016-10192.yml

```
@@ -268,8 +302,11 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: true
+  note: |
+    The fact that ffserver is a mutlimedia server used to broadcast live videos 
```

Typo of "multimedia"
```suggestion
    The fact that ffserver is a multimedia server used to broadcast live videos 
```